### ISSUE TITLE
mAP issue

### ISSUE THREAD

**Issue by ayushchopra96** [weiliu89/caffe#371]
Please use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) for usage, installation, or modeling questions, or other requests for help.
_Do not post such requests to Issues._ Doing so interferes with the development of Caffe.

Please read the [guidelines for contributing](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md) before submitting this issue.

### Issue summary
after 10000 iterations , it shows mAP as 0.66 , trained on the VOC data itself. Pls help explaining.

### Steps to reproduce

If you are having difficulty building Caffe or training a model, please ask the caffe-users mailing list. If you are reporting a build error that seems to be due to a bug in Caffe, please attach your build configuration (either Makefile.config or CMakeCache.txt) and the output of the make (or cmake) command.

### Your system configuration
Operating system:
Compiler:
CUDA version (if applicable):
CUDNN version (if applicable):
BLAS:
Python or MATLAB version (for pycaffe and matcaffe respectively):

**Comment by weiliu89** [comment]
In the README.md, it says the following:

```
# It should reach 77.* mAP at 120k iterations.
```

It is already good after 10k iterations, but needs much longer to converge to the best performance.

**Comment by ayushchopra96** [comment]
Sorry for the naive question but does 0.66 mean 66% result? Please pardon the very stupid doubt :)

**Comment by wk910930** [comment]
@ayushchopra96 Yes, 0.66 means 66%.

**Comment by virtuoso108** [comment]
I am getting 56mAP after 120k iterations on PASCAL VOC0712 data I am doing more iterations after this to improve the accuracy but it still stays at 56.*mAP. I used  batch_size = 1, num_gpus = 1 and base_lr = 0.00004. What can I try to improve the performance?

**Comment by weiliu89** [comment]
You need to adjust learning rate etc for different batch_size and num_gpus.
